lang/lua5.4/Makefile

```diff
@@ -0,0 +1,164 @@
+#
+# Copyright (C) 2006-2014 OpenWrt.org
```

Comment:
Since this is a new package, shouldnt you update the copyright bcs you are affiliated with OpenWrt?